host_guest/cyclodextrin_derivatives/guest_files/README.md

```diff
@@ -1,10 +1,11 @@
 # Guests for the SAMPL7 Cyclodextrin Derivatives Challenge
 
-Guests are trans-3-methylcyclohexanol and R-rimantadine hydrochloride, codenamed `g1` and `g2` respectively.
+Provided are the two guests for this challenge in MOL2 (.mol2), PDB (.pdb) and SDF (.sdf) format. Guests are trans-3-methylcyclohexanol and R-rimantadine hydrochloride, codenamed `g1` and `g2` respectively.
```

Comment:
I would say that they are provided as SMILES strings, and then mol2, PDB and sdf files generated from those SMILES strings are also provided here. (Sometimes it is important that people know what the original source files are. For example, if you were a future user and you found one of the guests was wrong, you would want to know which file you should change in order to re-generate all of the other files.)

Reply:
Changed.